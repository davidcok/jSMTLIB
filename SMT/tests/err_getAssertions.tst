; tests get-assertions with illegal arguments
(set-option :interactive-mode true)
(set-logic QF_UF)
(get-assertions 1)
