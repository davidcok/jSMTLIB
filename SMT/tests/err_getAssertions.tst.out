success
success
(get-assertions 1)
                ^
(error "A get-assertions command takes no arguments")
